pep-0685.rst

```diff
@@ -59,7 +59,7 @@ via::
     re.sub(r'[^A-Za-z0-9-.]+', '_', name).lower()
 
 The use of an underscore/``_`` differs from PEP 503's use of a hyphen/``-``,
-and it also normalizes characters outside of those allowed by :pep`508`.
+and it also normalizes characters outside of those allowed by :pep:`508`.
```

Comment:
Oops, I didn't catch this; surprised it didn't result in a Sphinx-level syntax error. I could have the linters check for cases like this, but I'm not sure if its common or general enough to justify.